php/Blockonomics.php

```diff
@@ -217,9 +227,16 @@ private function set_headers($api_key)
         }
     }
 
+
     public function testSetup()
     {
-        $response = $this->get_callbacks();
+        $BCH_Enabled = get_option('blockonomics_bch');
```

Comment:
Use standard PHP conventions
- camelCase or under_score variables and function names. 
- Constants can be UPPERCASE. 
- Classes can be PascalCase

Reply:
Noted, will remember to it going forward. Thanks for the pointer!

Reply:
Better to change `$BCH_Enabled` to `$bch_enabled` now so we will not have to revisit this.

Reply:
done